lib/dishes.rb

```diff
@@ -0,0 +1,10 @@
+module Dishes
```

Comment:
Does Dishes need to be a separate class from Menu? Do they have different responsibilities?